apps/vault/contracts/Vault.sol

```diff
@@ -9,6 +9,14 @@ import "@aragon/os/contracts/lib/token/ERC20.sol";
 contract Vault is EtherTokenConstant, AragonApp, DepositableStorage {
     bytes32 public constant TRANSFER_ROLE = keccak256("TRANSFER_ROLE");
 
+    string private constant ERROR_DATA_NON_ZERO = "VAULT_DATA_NON_ZERO";
+    string private constant ERROR_NOT_DEPOSITABLE = "VAULT_NOT_DEPOSITABLE";
+    string private constant ERROR_DEPOSIT_VALUE_ZERO = "VAULT_DEPOSIT_VALUE_ZERO";
+    string private constant ERROR_TRANSFER_VALUE_ZERO = "VAULT_TRANSFER_VALUE_ZERO";
+    string private constant ERROR_VALUE_MISMATCH = "VAULT_VALUE_MISMATCH";
+    string private constant TOKEN_TRANSFER_FROM_REVERTED = "VAULT_TOKEN_TRANSFER_FROM_REVERT";
+    string private constant TOKEN_TRANSFER_REVERTED = "VAULT_TOKEN_TRANSFER_REVERTED";
```

Comment:
These last two should have the `ERROR_` prefix :).

Reply:
I was not using the prefix when it's a `_REVERT`, but I'll add it.